thirdparty/github.go

```diff
@@ -48,7 +48,7 @@ type GithubPatch struct {
 	Author         string `bson:"author"`
 	AuthorUID      int    `bson:"author_uid"`
 	MergeCommitSHA string `bson:"merge_commit_sha"`
-	CommitTitle    string `bson:"commit_title"`
+	CommitTitle    string `bson:"commit_title"` // named poorly, it's actually the commit message
```

Comment:
loving this honesty

Reply:
this is like noticing a painting you hung is crooked on a wall but you're not allowed to change it because it's at your grandma's house and she likes everything to stay exactly the same 😭